Formula/shogun.rb

```diff
@@ -0,0 +1,118 @@
+class Shogun < Formula
+  desc "Large scale machine learning toolbox"
+  homepage "http://www.shogun-toolbox.org/"
+  url "http://shogun-toolbox.org/archives/shogun/releases/6.1/sources/shogun-6.1.3.tar.bz2"
+  sha256 "57169dc8c05b216771c567b2ee2988f14488dd13f7d191ebc9d0703bead4c9e6"
+
+  depends_on "cmake" => :build
+  depends_on :java => ["1.7+", :build]
+  depends_on "pkg-config" => :build
+  depends_on "swig" => :build
+  depends_on "arpack"
+  depends_on "eigen"
+  depends_on "glpk"
+  depends_on "hdf5"
+  depends_on "json-c"
+  depends_on "lapack"
+  depends_on "lzo"
+  depends_on "nlopt"
+  depends_on "python" if MacOS.version <= :lion
+  depends_on "readline"
+  depends_on "snappy"
+  depends_on "xz"
+
+  # fixes the linking of the python inteface
+  # based on the patch:
+  # https://github.com/shogun-toolbox/shogun/commit/ff8840ce0e0091d7619dacdf84819ea19872baf4
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/9df360c/shogun/fix_python_linking.patch"
+    sha256 "9c7b34b53ab6f54129c570c682b4a3fcae78418b211b7a8607db9b44d4daad35"
+  end
+
+  # fixes when Accelerator framework is trying to be used as a LAPACK backend for Eigen
+  # cmake swallows some of the include header flags hence on some versions of OSX
+  # the include of <vecLib/cblas.h> will fail
+  # based on the patch:
+  # https://github.com/shogun-toolbox/shogun/commit/6db834fb4ca9783b6e5adfde808d60ebfca0abc9
```

Comment:
missing the date

Reply:
date? shouldn't be the commit date be enough of the time when this has been actually added, i.e. `git blame` or `git log`? :)

Reply:
```
Upstream commit from 30 Jan 2018 https://github.com/shogun-toolbox/shogun/commit/6db834fb4ca9783b6e5adfde808d60ebfca0abc9
```
would be the usual comment